### PR TITLE
Updated Meta tags and favicon

### PR DESCRIPTION
**Pull request by helloshantanu**
Updated Meta Tags and Favicon for the App.

Used Standard SEO Techniques to Boost site performance on Search.

I Have Added Your Email in meta tags as meet59patel@gmail.com, if this ismt your email, please update and Merge request

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,13 +2,53 @@
...
+    <!-- Web page description -->
+
+    <meta property="og:url" content="https://schooler.vercel.app" />
+    <!-- Url to this web page -->
+    <meta property="og:locale" content="en_GB" />
+    <!-- Main Language -->
+
+    <meta property="og:country-name" content="INDIA" />
+    <!-- Country -->
+    <meta property="og:email" content="meet59patel@gmail.com" />
```
Comment (meet59patel) [inline public/index.html:50]: My email is patelmeet59@gmail.com
Commit the change please.

**Review by meet59patel — CHANGES_REQUESTED** [review]
Just update email and I'll merge.

**Comment by meet59patel** [comment]
@helloshantanu Thanks a lot for the PR. But keep in mind that you won't be getting SLoP points for contributions to any project other than your assigned one. 
Good work on this.

**Comment by helloshantanu** [comment]
My project hasn't opened any issue yet, so hanging around contributing to others😅

**Comment by meet59patel** [comment]
> My project hasn't opened any issue yet, so hanging around contributing to others😅

Alright, makes sense. I just had to clarify about SLoP points though.

**Comment by meet59patel** [comment]
@helloshantanu looks like there's somewhere a little typo that is breaking whole site. Check out the deployment of your PR at https://schooler-eafi0wnhv.vercel.app/  (from View deployment button above)
image

The entire site just loads blank.

**Comment by helloshantanu** [comment]
Fixing it right away

**Comment by meet59patel** [comment]
@helloshantanu Looks like the site breaking issue is fixed 👍🏻
But in the latest commit you've added `package-lock.json` file, which shouldn't be there actually as we're using `Yarn` for this project. So `yarn.lock` file is already there. This may create issues with some of the deployment platforms. Can you simply delete `package-lock.json` and commit? I'll be able to merge after that. 

Great work otherwise 👍🏻

**Comment by helloshantanu** [comment]
Ok, will do that

**Review by meet59patel — APPROVED** [review]
LGTM 👍🏻

**Comment by meet59patel** [comment]
Fixes #5

**Pull request merged**